visualization/index.html

```diff
@@ -135,39 +135,39 @@ <h1>
 			<section id="legend" class="container">
 				<div id="dv_infobox" class="row">
 					<div class="column_3">
-						<h2>Cómo leer la visualización</h2>
+						<h2>How to read the display</h2>
 						<p>
-							La visualización muestra los artículos que identificamos como relacionados con la currícula del tercer ciclo de educación básica integrada (ex ciclo básico) de Uruguay.
+							The visualization shows the articles that we identify as related to the curriculum of the third cycle of integrated basic education (former basic cycle) of Uruguay.
```

Comment:
we may remove "(former basic cycle)" since it does not add context for English-speaking users

Reply:
Done.